include/oneapi/dpl/internal/dynamic_selection_impl/round_robin_policy_impl.h

```diff
@@ -0,0 +1,128 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ROUND_ROBIN_POLICY_IMPL_H
+#define _ROUND_ROBIN_POLICY_IMPL_H
+#pragma once
+
+#include <atomic>
+#include <ostream>
```

Comment:
again this kind of headers looks for debug purposes only. I propose either remove it or surround with the dedicated macro (as well as operator<< itself)